contents/javascript/VueJs/Intro.md

```diff
@@ -0,0 +1,133 @@
+# VueJs
+
+Author: [Lu Lechuan](https://github.com/LuLechuan)
+
+Vue.js is an open-source progressive [JavaScript framework](https://en.wikipedia.org/wiki/JavaScript_framework) for building user interfaces. It is designed to improve code quality and maintainablity. Simply put, it is used as a tool to write JavaScript fast and easy.
+
+## Advantages of VueJs
+
+VueJs is an approachable, versatile and performant framework that helps to create a more maintainable and testable database.
+
+* Approachable: VueJs is very easy to learn. Compared to other framework such as Augular, Vue is simple in term of API and design. Learning enough to build non-trivial applications typically takes less than a day.
+
+* Progressive: Vue is designed from the ground up to be incrementally adoptable. The core library is focused on the view layer only, and is easy to pick up and integrate with other libraries or existing projects. This means that if you have a large application, you can plug Vue into just a part of your application without disturbing the other components. A quote from Evan You - the owner of Vue is as follow:
+"Vue.js is a more flexible, less opinionated solution ( than Angular ). That allows you to structure your app the way you want it to be, instead of being forced to do everything the Angular way. It’s only an interface layer so you can use it as a light feature in pages instead of a full blown SPA."
```

Comment:
markdown allows you to format the quote like this 

> Some quote formatting
 that you can use 
 to make the text
> look good